polymode-export.el

```diff
@@ -428,6 +428,7 @@ for each polymode in CONFIGS."
      ("docbook"	"xml"  "DocBook XML" "docbook")
      ("opendocument"    "xml"	"OpenDocument XML" "opendocument")
      ("odt"     "odt"  "OpenOffice text document" "odt")
+     ("pdf"     "pdf"  "Portable Document Format" "latex")
```

Comment:
Are you shure about the last "latex"? It is replaced as -t option in pandoc command. Should be 'pdf' right?

Reply:
It is a bit mysterious, which is why I requested another person to check.

## With "pdf"
```
("pdf"     "pdf"  "Portable Document Format" "pdf")
```
I get this when I export:
```
Exporting markdown-->pdf with command:

     pandoc Selle_2019_Review.md -f markdown -t pdf -o Selle_2019_Review[exported].pdf

To create a pdf using pandoc, use -t latex|beamer|context|ms|html5
and specify an output file with .pdf extension (-o filename.pdf).
```

## Changing last "pdf" to  "latex" 
```
("pdf"     "pdf"  "Portable Document Format" "latex")
```
I get this when I export:
```
Exporting markdown-->pdf with command:

     pandoc Selle_2019_Review.md -f markdown -t latex -o Selle_2019_Review[exported].pdf

```

Reply:
Clear. Looks like pandoc uses a two step procedure internally, which makes sense indeed. Thanks!